frontend/package.json

```diff
@@ -22,6 +22,8 @@
     "@types/testing-library__jest-dom": "^5.14.5",
     "@uswds/uswds": "^3.8.0",
     "apollo-upload-client": "^17.0.0",
+    "axios": "^1.7.2",
```

Comment:
same question here - what's axios for?

Reply:
See reference below that explains vulnerability:
- [axios](https://app.snyk.io/org/prime-simplereport/project/9b116ed0-0b91-4572-ad4d-dac6bad2c3c5#issue-SNYK-JS-AXIOS-6032459)

Reply:
Not sure if this is what you were asking @mehansen but did a `yarn why axios` and looks like `smartystreets-javascript-sdk` depends on it
<img width="749" alt="Screenshot 2024-07-19 at 11 42 50" src="https://github.com/user-attachments/assets/eb59d936-6f56-4bb9-80f9-fcaf156719a6">